### DISCUSSION TITLE
Connect to The Things Network | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#731] [Q&A]
# Connect to The Things Network | Seeed Studio Wiki

Connect_to_The_Things_Network

https://wiki.seeedstudio.com/SenseCAP_T1000_tracker_TTN/

**Comment by moonlight0551** [comment]
In the section re Adding A Sensor to TTN Console, some of the screen shot images are missing and shown as icons. It would be very helpful to see actual screenshots of the 'Provisioning Information ie Join EUI and Device EUI and where they can be found in the SenseCraft app.

**Reply by GeoCoders1974** [reply]
Did you ever get this to work?

**Reply by moonlight0551** [reply]
Yes eventually. Used the SenseCAP Sense app on my android. Connected to the sensor via bluetooth. Found for Australia TTN frequency setup, I had to select Other (Not TTN) whic opened up frequency band option and more importantly a selection for a sub band which for AUS is 2 otherwise the device never sends its join requent on the right frequency and can never join. Also you can now assign the TTN keys. These are not shown otherwise.

**Comment by GeoCoders1974** [comment]
Probably the worst instructions ever. Major jumps, missing information, etc. For such a great idea, great price, the product backend is horrible.

**Reply by Jessie219-web** [reply]
Hi there, thanks for your feedback, we will update this wiki to make it more detailed.

**Reply by HaViGit** [reply]
Indeed, very bad instructions for an over 300 euros product. As mentioned by GeoCoders1974: Horrible!

**Reply by HaViGit** [reply]
@GeoCoder1974: Did you finally succeed? Can you advice me?

**Reply by GeoCoders1974** [reply]
I did, but by trial and error. I wouldnt be much help.

**Comment by bkonnet** [comment]
Hi. I´m triying to set it in TTN. AU915, but if select it from the Lorawan repository there is not a way to use FSB1 or any other channel.  How can this be done?
Also the decoder that is in the github repository is too long and it can not be used as is in TTN.

**Reply by nfs0619** [reply]
Hi, we recommend reaching out to our technical support team at [sensecap@seeed.cc](mailto:sensecap@seeed.cc) with your questions and suggestions for further assistance. Regarding your concern about the decoder being too long, we will consider optimizing it. Additionally, you can check our forum at [https://forum.seeedstudio.com/](https://forum.seeedstudio.com/) to see if other users have shared similar experiences. Thanks for your feedback!

**Comment by Pixeldieb** [comment]
Would love to see an Updated Chirpstack V4 in the Repository.
Submitted a simple conversion with Gemini AI.

**Reply by Jessie219-web** [reply]
you mean the decoder of Chirpstack V4? it should be the same as the TTN decoder: https://github.com/Seeed-Solution/SenseCAP-Decoder/blob/main/T1000/TTN/SenseCAP_T1000_TTN_Decoder.js
